### DISCUSSION TITLE
Hola Buenos Dias Alguien Me Podria Decir como van Ordenadas Las Practicas

### DISCUSSION THREAD

**Discussion by DyFrabian** [Launch-X-Latam/MisionFrontEnd#210] [Q&A]
Hola Buenos Dias , Alguien me podria decir como van ordenadas las practicas .

**Comment by doguedogue** [comment]
asi mira, puedes ponerlo toda esa práctica 1  en un repo, la 2 en otro y asi:
Prática 1
https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/01%20-%20INTRO/practicas

![image](https://user-images.githubusercontent.com/23409026/196052437-1beea5d1-4552-47b1-a710-6b62c4816453.png)

Prática 2
https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/02%20-%20HTML/practicas
